docs/operating-scylla/nodetool.rst

```diff
@@ -67,19 +67,6 @@ The ``nodetool`` utility provides a simple command-line interface to the followi
 
 Nodetool generic options
 ========================
-All options are supported:
```

Comment:
Same as --port, I don't understand why it being removed.
Seems like these options are exist on native nodetool:
https://github.com/scylladb/scylladb/blob/master/tools/scylla-nodetool.cc#L3206

Reply:
Ah okay, then maybe we don't need to describe other options.